tests/basic/pool_function_test.go

```diff
@@ -859,3 +860,63 @@ var _ = Describe("{PoolExpandAddDiskInMaintenanceMode}", func() {
 		})
 	})
 })
+
+var _ = Describe("{StorageFullPoolExpansion}", func() {
+	var (
+		appList      []string
+		selectedNode *node.Node
+	)
+
+	BeforeEach(func() {
+		Inst().AppList = []string{"fio-fastpath-repl1"}
+		contexts = ScheduleApplications("storagefull-resize")
+		appList = Inst().AppList
+	})
+
+	JustBeforeEach(func() {
+		selectedNode = GetNodeWithLeastSize()
+		_ = Inst().S.AddLabelOnNode(*selectedNode, k8s.NodeType, k8s.FastpathNodeType)
```

Comment:
I am not aware. What happens with this label?

Reply:
This makes the testing app(fio-fastpath) to write to a pool on this specific node